### ISSUE TITLE
Modpack update process broken

### ISSUE THREAD

**Issue by lhw** [gorilla-devs/GDLauncher#843]
**Describe the bug**
Updating an existing modpack right now via Manage>Modpack kicks off the update process, at least in the UI but then stops at 100% downloading game files. The mods directory is cleared but no mods were downloaded in the process. resulting in a broken modpack. Installer lock remains. No further actions are possible with the modpack until a gdlauncher restart.

**To Reproduce**
1. Select modpack to update. Go to Manage > Modpacks
2. Select new version and click switch version
3. ...
4. Profit

**Expected behavior**
Works as before

**Possible solutions**
The Stacktrace is not all that helpful. I'm guessing a parsed page might've been changed on curseforge.

**Screenshots**
![image](https://user-images.githubusercontent.com/351875/115399309-58d87f00-a1e8-11eb-91c3-ea3f4e6f2db1.png)
![image](https://user-images.githubusercontent.com/351875/115399383-6c83e580-a1e8-11eb-9ff1-a05da16c4d91.png)

**Operating System:**
 - OS: Ubuntu
 - Version: 20.10

**Additional context**
Console log:
```
index.html#/home:1 A cookie associated with a cross-site resource at http://minecraft.net/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
webpack-internal:///1028:264 setup
background.png:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
DevTools failed to parse SourceMap: file:///opt/GDLauncher/resources/app.asar/build/static/css/5.717edb1f.chunk.css.map
DevTools failed to parse SourceMap: file:///opt/GDLauncher/resources/app.asar/build/static/css/6.0ae82f3c.chunk.css.map
actions.js:649 Error: EEXIST: file already exists, open '/home/lhw/.config/gdlauncher_next/instances/ATM6 To The Sky/installing.lock'
eval @ actions.js:649
notStale @ lockfile.js:247
eval @ lockfile.js:186
(anonymous) @ fs.js:146
asyncToGenerator.js:6 Uncaught (in promise) TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:112)
    at Object.join (path.js:1040)
    at _callee28$ (actions.js:649)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:293)
    at Generator.eval [as next] (runtime.js:118)
    at asyncGeneratorStep (asyncToGenerator.js:3)
    at _next (asyncToGenerator.js:25)
    at eval (asyncToGenerator.js:32)
    at new Promise (<anonymous>)
NodeError @ internal/errors.js:216
validateString @ internal/validators.js:112
join @ path.js:1040
_callee28$ @ actions.js:649
tryCatch @ runtime.js:63
invoke @ runtime.js:293
eval @ runtime.js:118
asyncGeneratorStep @ asyncToGenerator.js:3
_next @ asyncToGenerator.js:25
eval @ asyncToGenerator.js:32
eval @ asyncToGenerator.js:21
eval @ actions.js:1088
eval @ thunkEnhancer.js:1
dispatch @ redux.js:638
_callee31$ @ actions.js:1290
tryCatch @ runtime.js:63
invoke @ runtime.js:293
eval @ runtime.js:118
asyncGeneratorStep @ asyncToGenerator.js:3
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
eval @ asyncToGenerator.js:32
eval @ asyncToGenerator.js:21
eval @ actions.js:1290
eval @ thunkEnhancer.js:1
dispatch @ redux.js:638
_callee22$ @ actions.js:649
tryCatch @ runtime.js:63
invoke @ runtime.js:293
eval @ runtime.js:118
asyncGeneratorStep @ asyncToGenerator.js:3
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
eval @ asyncToGenerator.js:32
eval @ asyncToGenerator.js:21
eval @ actions.js:649
eval @ thunkEnhancer.js:1
dispatch @ redux.js:638
_callee34$ @ actions.js:1290
tryCatch @ runtime.js:63
invoke @ runtime.js:293
eval @ runtime.js:118
asyncGeneratorStep @ asyncToGenerator.js:3
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
Promise.then (async)
asyncGeneratorStep @ asyncToGenerator.js:13
_next @ asyncToGenerator.js:25
eval @ asyncToGenerator.js:32
eval @ asyncToGenerator.js:21
eval @ actions.js:1290
eval @ thunkEnhancer.js:1
_callee3$ @ Modpack.js:1
tryCatch @ runtime.js:63
invoke @ runtime.js:293
eval @ runtime.js:118
asyncGeneratorStep @ asyncToGenerator.js:3
_next @ asyncToGenerator.js:25
eval @ asyncToGenerator.js:32
eval @ asyncToGenerator.js:21
handleClick @ button.js:236
ba @ react-dom.production.min.js:14
ja @ react-dom.production.min.js:14
ka @ react-dom.production.min.js:14
oa @ react-dom.production.min.js:15
lc @ react-dom.production.min.js:52
jc @ react-dom.production.min.js:51
mc @ react-dom.production.min.js:52
sc @ react-dom.production.min.js:56
Ia @ react-dom.production.min.js:287
Ma @ react-dom.production.min.js:19
Rc @ react-dom.production.min.js:70
id @ react-dom.production.min.js:69
exports.unstable_runWithPriority @ scheduler.production.min.js:19
cg @ react-dom.production.min.js:122
Ga @ react-dom.production.min.js:287
gd @ react-dom.production.min.js:68
```

**Comment by Irritated-Badger** [comment]
Same issue, Windows 10. I don't have a log because I deleted the instance figuring it was an issue on my end. Then I ran into the whole, can't create a new instance, issue.

**Comment by TOLoneWolf** [comment]
When updating it has to install forge and it's the forge format data that chanced breaking new installs of forge ( breaking updating as well)

#841

**Comment by Charismara** [comment]
I got the same problem. Any information about a fix / workaround?

**Comment by blarfoon** [comment]
It should be fixed with the new release (1.1.1). Feel free to reopen if the issue persists

**Comment by lhw** [comment]
Is there a lock that stops the the updater from re-downloading mods? The update process did not download any mods when switching versions. Creating a new instance worked just fine, including downloading mods.
